src/assets/images/MetaMaskLogo.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+interface Props {
+    className?: string;
+    onClick?: () => void;
+}
+
+export const MetaMaskLogo: React.FC<Props> = (props: Props) => {
+    return (
+        <div className={props.className} onClick={props.onClick}>
+            <svg width="35" height="33" viewBox="0 0 35 33" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M32.9583 1L19.8242 10.7183L22.2666 4.99099L32.9583 1Z" fill="#E17726" stroke="#E17726" stroke-width="0.25" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Use color env for fill and stroke attributes

Reply:
no need, it't metaMask icon😂

Reply:
The customer can ask us to change the appearance of this icon, so be good to make an additional color envs for this icon if it's impossible to use existing ones